memorix-recordtypes/Archiefblok.ttl

```diff
@@ -4,9 +4,9 @@
 @prefix xsd:           <http://www.w3.org/2001/XMLSchema#> .
 
 @prefix rt:            </resources/recordtypes/> .
-@prefix fonds:         </resources/recordtypes/Fonds#> .
+@prefix fonds:         </resources/recordtypes/Fondsje#> .
```

Comment:
is het echt Fondsje nu?

Reply:
Ai, Nee natuurlijk niet 😊  ... Oeps van mijn kant om te kijken of het werkte (ik kon die andere niet weggooien natuurlijk). Ga ik meteen fiksen